library/xml-model/src/main/java/org/cru/godtools/xml/model/Text.kt

```diff
@@ -44,18 +45,28 @@ class Text : Content {
     private val _textScale: Double?
     val textScale get() = _textScale ?: DEFAULT_TEXT_SCALE
 
+    private val imageStartName: String?
+    val imageStart get() = getResource(imageStartName)
+    private val imageEndName: String?
+    val imageEnd get() = getResource(imageEndName)
+
+
     @RestrictTo(RestrictTo.Scope.TESTS)
     constructor(
         parent: Base,
         text: String? = null,
         textScale: Double? = null,
         @ColorInt textColor: Int? = null,
-        textAlign: Align? = null
+        textAlign: Align? = null,
+        imageStart: String? = null,
+        imageEnd: String? = null
     ) : super(parent) {
         this.text = text
         _textAlign = textAlign
         _textColor = textColor
         _textScale = textScale
+        this.imageStartName = imageStart
+        this.imageEndName = imageEnd
```

Comment:
no need for `this.` on these because there isn't ambiguity between the properties and constructor params